domain/infosync/tiflash_manager.go

```diff
@@ -405,6 +442,58 @@ func (tiflash *MockTiFlash) HandleGetPDRegionRecordStats(_ int64) helper.PDRegio
 	}
 }
 
+// HandleGetPDInvalidRegionsInfo is mock function for GetPDInvalidRegionsInfo.
+// It currently always returns 1 region with 1 down-peer Region and 1 pending peer
```

Comment:
Why is it a mock function?

Reply:
The interface in `MockTiFlash ` must be mock function.
If we define a function in `tiflash_manager` then need define a mock function in here.

Reply:
ok